contracts/util/ZeroCollateralCommon.sol

```diff
@@ -2,17 +2,27 @@ pragma solidity 0.5.17;
 
 
 library ZeroCollateralCommon {
-    // Interest accrued from lending account
-    struct LendAccount {
-        uint256 lastBlockAccrued;
-        uint256 totalAccruedInterest;
-    }
-
     // Borrower account details
     struct Borrower {
         uint256 lastBorrowId;
     }
 
+    // The amount of interest owed to a borrower
+    // The interest is just that accrued until `blockLastAccrued`
+    struct AccruedInterest {
+        uint256 totalAccruedInterest;
+        uint256 totalNotWithdrawn;
+        uint256 blockLastAccrued;
+    }
+
+    struct AggregatedInterest {
+        uint256 totalSubmissions;
+        uint256 maxValue;
+        uint256 minValue;
+        uint256 sumOfValues;
+        bool finalised;
```

Comment:
Would it be 'finalized'?

Reply:
Ah yes. `finalised` is the British spelling. I will change it!